src/python/pants/backend/python/dependency_inference/module_mapper_test.py

```diff
@@ -415,12 +415,14 @@ def test_map_module_to_address(rule_runner: RuleRunner) -> None:
     def assert_owners(
         module: str, expected: list[Address], expected_ambiguous: list[Address] | None = None
     ) -> None:
-        owners = rule_runner.request(PythonModuleOwners, [PythonModule(module)])
+        owners = rule_runner.request(
+            PythonModuleOwners, [PythonModuleOwnersRequest(module, resolve=None)]
```

Comment:
I should probably update this test to include `resolve` stuff...The goal is for _every_ callsite to eventually have to set the resolve.

I only didn't because the test is already so complex. I guess at a minimum this should set the `resolve` to `[python].default_resolve`? And then possibly have a new, more focused test with edge cases of `resolve`?

Reply:
Yea, reasonable.